src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue27418.cs

```diff
@@ -0,0 +1,25 @@
+#if TEST_FAILS_ON_CATALYST
+// When maximizing the window to verify the screenshot on Mac, resizing did not occur properly on cv1. Therefore, I have restricted Catalyst for now.
```

Comment:
Can re-enable the Mac test after merge https://github.com/dotnet/maui/pull/27531